composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1|^8.0",
         "ext-mongodb": "^1.2.0",
         "mongodb/mongodb": "^1.1.0"
     },
     "require-dev": {
         "symfony/var-dumper": "^3.3",
-        "phpunit/phpunit": "^6.4"
+        "phpunit/phpunit": "~6.4"
```

Comment:
Is there any reason not to stick with the `^` operator ?

Reply:
Yes, I think it could also be specified like this (same as with PHP):

` "phpunit/phpunit": "^6.4|^7.0|^8.0|^9.0"`

I honestly don't know what's best in these situations. 
It should be mentioned that this change is not required to make the package work on PHP8, this is only to allow the tests to run on modern versions of PHPUnit. I just noticed this because I wanted to run the test suite on my PHPUnit 9.5.2.
I can leave it as-is if you prefer.

Reply:
In summary:
the "issues" are in fact only with `"require-dev"`. PHP8 requires PHPUnit 8 or higher. Higher version of PHPUnit requires the changes to the `setUp` and `tearDown` methods to match the method signature, which, in turn, requires at minimum a PHP 7.1 version to be valid syntax.
I'm not entirely familiar with how this is done otherwise, but I guess it would be valid to specify a different PHP version requirement under `"require-dev"`?